charts/nginx/templates/nginx-service.yaml

```diff
@@ -13,6 +13,11 @@ metadata:
     heritage: {{ .Release.Service }}
   annotations:
     service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
```

Comment:
I feel like we should take this out and tell customers who need it to add it. This was very recently added.

Reply:
@ianstanton ^^^ ?

Reply:
@danielhoherd I agree with this. Keeps things generic and gives customers the liberty to add this if they want. We'll need to add some instructions for this. @jwitz Who is in charge of document updates these days?

Reply:
what if service.beta.kubernetes.io/aws-load-balancer-type: "nlb" was removed from here and added a a default helm chart value as both documentation as code and to keep that default in place?

Reply:
Didn't like my other thought. Would like to see each of the deployment sections in the docs have common annotations for each cloud service noted there for customers to copy and remove mention of private from the sample yaml in favor of this method.

Reply:
@rbankston if you haven't yet, you may want to turn that into a ticket so the idea doesn't get lost when i resolve this conversation.